test/instance.js

```diff
@@ -1015,12 +1015,15 @@ describe('Bigtable/Instance', function() {
 
   describe('setMetadata', function() {
     it('should provide the proper request options', function(done) {
-      var metadata = {a: 'b'};
-      var expectedMetadata = extend({name: instance.id}, metadata);
+      var metadata = {displayName: 'updateDisplayName'};
+      var expectedMetadata = {
+        instance: extend({name: instance.id}, metadata),
```

Comment:
This is somewhat of a testing anti-pattern, just create the object that you expect, in this case  

```js
{
  name: instance.name,
  metadata: metadata,
}
```